alembic/script/write_hooks.py

```diff
@@ -176,3 +177,36 @@ def exec_(path: str, options: dict, ignore_output: bool = False) -> None:
         cwd=cwd,
         **kw,
     )
+
+
+@register("module")
+def module(path: str, options: dict, ignore_output: bool = False) -> None:
```

Comment:
This is just a lot of code.  Can this be consolidated with "console_scripts" into a common function since these are 90% the same ?

Reply:
same for `exec_`, I could try pull out the common parts